js/main.js

```diff
@@ -0,0 +1,16 @@
+//Функция, возвращающая случайное целое число из переданного диапазона включительно,источник https://schoolsw3.com/js/js_random.php
+function getRandomFloor(min, max) {
```

Comment:
Лучше подправить нейминг, getRandomFloor не совсем подходит для целого числа. Предлагаю заменить на 
```suggestion
function getRandomInteger(min, max) {
```